xml/art_admin_slemicro.xml

```diff
@@ -1332,11 +1327,771 @@ After=etcd.service
+    <varlistentry>
+     <term><replaceable>PCP_CONTAINER_IMAGE</replaceable></term>
+     <listitem>
+      <para>
+       Is the downloaded PCP container image.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+   <para>
+    Other usefull options of the <command>podman run</command> command follow:
+   </para>
+   <variablelist>
+    <title>Other options</title>
+    <varlistentry>
+     <term><option>-p <replaceable>HOST_IP:HOST_PORT:CONTAINER_PORT</replaceable></option></term>
+     <listitem>
+      <para>
+       Publishes ports of the container by mapping a container port onto a host
+       port. If you don't specify <replaceable>HOST_IP</replaceable>, the ports
+       will be mapped on the local host. If you omit the
+       <replaceable>HOST_PORT</replaceable> value, a random port number will be
+       used. By default the <command>pmcd</command> daemon listens and exposes
+       the PMAPI to receive metrics on the port <emphasis>44321</emphasis>, so
+       it is recommended to map this port on the same port number on the host.
+       The <command>pmproxy</command> daemon listens on and exposes the REST
+       PMWEBAPI to access metrics on the <emphasis>44322</emphasis> port by
+       default, so it is recommended to map this port on the same host port
+       number.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><option>--net host</option></term>
+     <listitem>
+      <para>
+       The container uses the host's network. Use this option if you want to
+       collect metrics from the host's network interfaces.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><option>-e</option></term>
+     <listitem>
+      <para>
+       The option enables you to set the following environment variables:
+      </para>
+      <variablelist>
+       <!-- <title>Environment variables</title> -->
+       <varlistentry>
+        <term>PCP_SERVICES</term>
+        <listitem>
+         <para>
+          is a comma separated list of services to start by &systemd; in the
+          container.
+         </para>
+         <para>
+          Default services are: <literal>pmcd</literal>,
+          <literal>pmie</literal>, <literal>pmlogger</literal>,
+          <literal>pmproxy</literal>.
+         </para>
+         <para>
+          You can use this variable, if you want to run a container with other
+          than the default list of services, for example only with
+          <literal>pmlogger</literal>:
+         </para>
+<screen>&prompt.root;podman run -d \
+  --name pmlogger \
+  --systemd always \
+  -e PCP_SERVICES=pmlogger  \
+  -v pcp-archives:/var/log/pcp/pmlogger  \
+  registry.suse.com/suse/pcp:latest</screen>
+        </listitem>
+       </varlistentry>
+       <varlistentry>
+        <term>HOST_MOUNT</term>
+        <listitem>
+         <para>
+          is a path inside the container to the bind mount on the host. This
```

Comment:
`is the path inside the container to the bind mount of the host's root filesystem.`